packages/client/hmi-client/src/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -22,40 +22,52 @@
 				placeholder="Select a simulation run"
 				@update:model-value="handleSelectedRunChange"
 			/>
-			<tera-simulate-chart
-				v-if="runResults[selectedRun?.runId]"
-				:run-results="runResults[selectedRun.runId]"
-				:chartConfig="node.state.chartConfigs[selectedRun.idx]"
-				has-mean-line
-				@configuration-change="configurationChange(selectedRun.idx, $event)"
-			/>
-			<tera-dataset-datatable
-				v-if="rawContent[selectedRun?.runId]"
-				:rows="10"
-				:raw-content="rawContent[selectedRun.runId]"
-			/>
-			<Button
-				class="add-chart"
-				title="Saves the current version of the model as a new Terarium asset"
-				@click="showSaveInput = !showSaveInput"
-			>
-				<span class="pi pi-save p-button-icon p-button-icon-left"></span>
-				<span class="p-button-text">Save as</span>
-			</Button>
-			<span v-if="showSaveInput" style="padding-left: 1em; padding-right: 2em">
-				<InputText v-model="saveAsName" class="post-fix" placeholder="New dataset name" />
-				<i
-					class="pi pi-times i"
-					:class="{ clear: hasValidDatasetName }"
-					@click="saveAsName = ''"
-				></i>
-				<i
-					v-if="useProjects().activeProject.value?.id"
-					class="pi pi-check i"
-					:class="{ save: hasValidDatasetName }"
-					@click="saveDatasetToProject"
-				></i>
-			</span>
+			<template v-if="runResults[selectedRun?.runId]">
+				<tera-simulate-chart
+					v-for="(cfg, idx) in node.state.simConfigs.chartConfigs"
+					:key="idx"
+					:run-results="runResults[selectedRun.runId]"
+					:chartConfig="{ selectedRun: selectedRun.runId, selectedVariable: cfg }"
+					has-mean-line
+					@configuration-change="configurationChange(idx, $event)"
+				/>
```

Comment:
We will want to refactor this tera-simulate-chart as a subsequent task, it can not decide if it wants to operate on an array or operate on an object, and in turn that is confusing to whoever is using the component.